sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/AwsCloudProfile.cs

```diff
@@ -0,0 +1,84 @@
+        /// </para>
+        /// <para>
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="AwsCloudProfile"/>. </summary>
+        /// <param name="accountId"> Account id for the AWS account. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="accountId"/> is null. </exception>
+        public AwsCloudProfile(string accountId)
+        {
+            Argument.AssertNotNull(accountId, nameof(accountId));
+
+            AccountId = accountId;
+            ExcludedAccounts = new ChangeTrackingList<string>();
+        }
+
+        /// <summary> Initializes a new instance of <see cref="AwsCloudProfile"/>. </summary>
+        /// <param name="accountId"> Account id for the AWS account. </param>
+        /// <param name="excludedAccounts"> List of AWS accounts which need to be excluded. </param>
+        /// <param name="isOrganizationalAccount"> Boolean value that indicates whether the account is organizational or not. True represents organization account, whereas false represents a single account. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal AwsCloudProfile(string accountId, IList<string> excludedAccounts, bool? isOrganizationalAccount, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            AccountId = accountId;
+            ExcludedAccounts = excludedAccounts;
+            IsOrganizationalAccount = isOrganizationalAccount;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="AwsCloudProfile"/> for deserialization. </summary>
+        internal AwsCloudProfile()
+        {
+        }
+
+        /// <summary> Account id for the AWS account. </summary>
+        public string AccountId { get; set; }
```

Comment:
Is it a GUID?

Reply:
No, it's AWS accountId in 12-digit number